bindings/matrix-sdk-crypto-js/src/identities.rs

```diff
@@ -0,0 +1,172 @@
+//! User identities.
+
+use js_sys::{Array, Promise};
+use wasm_bindgen::prelude::*;
+
+use crate::{
+    future::future_to_promise, identifiers, impl_from_to_inner, js::try_array_to_vec, requests,
+    verification,
+};
+
+pub(crate) struct UserIdentities {
+    inner: matrix_sdk_crypto::UserIdentities,
+}
+
+impl_from_to_inner!(matrix_sdk_crypto::UserIdentities => UserIdentities);
+
+impl From<UserIdentities> for JsValue {
+    fn from(user_identities: UserIdentities) -> Self {
+        use matrix_sdk_crypto::UserIdentities::*;
```

Comment:
any reason this import isn't at the top?

Reply:
I don't want all its variants to be global to the module; just available locally for this function only.